.mailmap

```diff
@@ -1182,6 +1182,7 @@ Pastafarianist <mr.pastafarianist@gmail.com>
 Patrick Lacasse <patrick.m.lacasse@gmail.com>
 Patrick Poitras <acebulf@gmail.com>
 Paul Mandel <paulmandel@google.com>
+Paul Scofield <cerebral.paul@gmail.com>
```

Comment:
The [commit](https://github.com/sympy/sympy/commit/e4164b0b88610efb1df98791afbd0450177e6ec5.patch) has:
```
Paul Scofield <Cerebral.Paul@gmail.com>
```
You might need:
```
Paul Scofield <cerebral.paul@gmail.com> Paul Scofield <Cerebral.Paul@gmail.com>

```